test/jest-api.teardown-setup.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+const fs = require('fs');
+const Knex = require('knex');
+
+const databases = {
+  postgres: {
+    client: 'postgres',
+    connection: {
+      connectionString: 'postgresql://strapi:strapi@127.0.0.1',
+    },
+  },
+  mysql: {
+    client: 'mysql',
+    connection: {
+      host: '127.0.0.1',
+      port: 3306,
+      database: 'strapi_test',
+      username: 'root',
+      password: 'strapi',
+    },
+  },
+  sqlite: {
+    client: 'sqlite',
+    connection: {
+      filename: './tmp/data.db',
+    },
+    useNullAsDefault: true,
+  },
+};
+
+async function main(uuid) {
+  const knex = Knex(databases[process.env.DATABASE]);
+
+  // Lets create our database if it does not exist
+  await knex.raw(`DROP DATABASE ${uuid};`);
+  // knex.destory();
```

Comment:
missing coment :) , should we close the connection here though?

Reply:
knex should automaticly be garbage collected after this anyway since all refrence to the knex object is lost.